client/state/data-layer/wpcom-http/README.md

```diff
@@ -188,7 +188,7 @@ import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 dispatchRequest( fetchMenu, addMenuItems, alertFailure );
 
 // create request when no meta present, indicate on success, undo on failure, update on progress
-dispatchRequest( sendRecipe, indicateSuccess, removeRecipe, updateRecipeProgress );
+dispatchRequest( sendRecipe, indicateSuccess, removeRecipe, { onProgress: updateRecipeProgress } );
```

Comment:
oh yeah - I planned on making a smaller `onProgress` PR before-hand but I haven't done that yet